src/compose.ts

```diff
@@ -385,6 +402,35 @@ interface VolumeRef {
 	};
 }
 
+const appliedBindMountsByLabel: Array<[string, string[]]> = [
+	['io.balena.features.balena-socket', ['/var/run/docker.sock']],
+	['io.balena.features.balena-socket', ['/var/run/balena-engine.sock']],
+	['io.balena.features.dbus', ['/run/dbus']],
+	['io.balena.features.sysfs', ['/sys']],
+	['io.balena.features.procfs', ['/proc']],
+	['io.balena.features.kernel-modules', ['/lib/modules']],
+	['io.balena.features.firmware', ['/lib/firmware']],
+	[
+		'io.balena.features.journal-logs',
+		['/var/log/journal', '/run/log/journal', '/etc/machine-id'],
+	],
+];
+
+const bindMountTargets: Dict<string> = {
+	'/var/run/docker.sock': '/var/run/docker.sock',
+	'/var/run/balena-engine.sock': '/var/run/balena-engine.sock',
```

Comment:
`/host/run/balena-engine.sock` is used as the `DOCKER_HOST` 

https://github.com/balena-os/balena-supervisor/blob/27013b1d72f5cf66b0ae928b7e7e66bb34d7035d/src/compose/utils.ts#L357-L359

```suggestion
	'/var/run/balena-engine.sock': '/var/run/balena-engine.sock',
	'/var/run/balena-engine.sock': '/host/run/balena-engine.sock',
```

Reply:
This is interesting -- I think I should ignore the target path @pipex because otherwise we're back at the "not a single source of truth" problem. That is, source paths are enough to infer user intent, but target paths are an implementation detail. Does this sound right?

Reply:
Yes, I think that's correct